models/en-US.json

```diff
@@ -6,15 +6,24 @@
             "intents":[
                 {
                   "name": "AMAZON.CancelIntent",
-                  "samples": []
+                  "samples": [
+                    "cancel",
+                    "wait",
+                    "cancel hello world"
+                  ]
                 },
                 {
                   "name": "AMAZON.HelpIntent",
                   "samples": []
                 },
                 {
                   "name": "AMAZON.StopIntent",
-                  "samples": []
+                  "samples": [
```

Comment:
Why add these additional utterances? We add default samples for the builtin intents. You can see them here:
https://github.com/bespoken/virtual-alexa/blob/master/src/BuiltinUtterances.ts

It doesn't look like you use the custom utterances, so I would not include this. Or if you do include it, just include the ones that are custom (stop hello world, end, and end hello world).

The same applies for cancel.